### DISCUSSION TITLE
with Deeplink

### DISCUSSION THREAD

**Discussion by mrakramov** [bumble-tech/appyx#109] [Q&A] [answered]
I haven't found the documentation related to deeplink.
Can deeplink be used ?
if yes, could you show me some examples.

**Comment by KovalevAndrey** [comment, marked as answer]
Yes, documentation is not ready yet but we have samples of how to do that. You will need to use workflow API for that, check out this example https://github.com/bumble-tech/appyx/blob/main/sandbox/src/main/kotlin/com/bumble/appyx/sandbox/client/workflow/WorkflowExampleActivity.kt
